src/steps/doBackportVersions.ts

```diff
@@ -171,19 +175,26 @@ async function resolveConflictsOrAbort(owner: string, repoName: string) {
   }
 }
 
-function getPullRequestTitle(baseBranch: string, commits: Commit[]) {
-  const commitMessages = commits
+function getPullRequestTitle(
+  baseBranch: string,
+  commits: Commit[],
+  prTitle: string
+) {
+  // @ts-ignore
+  const commitMessages = commits // eslint-disable-line @typescript-eslint/no-unused-vars
     .map(commit => commit.message)
     .join(' | ')
     .slice(0, 200);
 
-  return `[${baseBranch}] ${commitMessages}`;
+  // prTitle could include baseBranch or commitMessages in template literal
+  return eval('`' + prTitle + '`');
```

Comment:
I think it's better to avoid `eval` and explicitly replacing certain tokens. Eg.
```js
prTitle
  .replace('${baseBranch}', baseBranch)
  .replace('${commitMessages}', commitMessages)
```

It's not as powerful but it's explicit and will make it more obvious when breaking the contract with the end user. Plus, I'm afraid of `eval` 😱

Would this solution still work for you?

Reply:
Yes, this can definitely still work. I was weary of using `eval` in the first place. Fixed in fe109c0

Reply:
Looks great!

I noticed that `$` is treated as a special character and everything after it will be stripped. So the following won't work: `--prTitle="Custom title: ${commitMessages}"`

This is a limitation with bash. Quick google turned up [this](https://superuser.com/questions/163515/bash-how-to-pass-command-line-arguments-containing-special-characters) and [this](https://unix.stackexchange.com/questions/270977/what-characters-are-required-to-be-escaped-in-command-line-arguments).

**Workarounds**
 - Using single quotes : `--prTitle='Custom title: ${commitMessages}'`
 - Escaping: `--prTitle="Custom title: \${commitMessages}"`

To bypass the problem altogether it might be better to drop the `$` and just use curly brackets - that way users won't have to resort to the workarounds above:
```js
prTitle
  .replace('{baseBranch}', baseBranch)
  .replace('{commitMessages}', commitMessages)
```

Reply:
Good catch, didn't think about that. Removed the dollar sign notation in 
8e9bbe5